### PR TITLE
Update the installation guidance and README.md

### PR DESCRIPTION
**Pull request by Jeffwan**
## Pull Request Description

1. Start to write the installation guidance
2. Update the README.md feature section

image

## Related Issues
Resolves: #[Insert issue number(s)]

**Important: Before submitting, please complete the description above and review the checklist below.**

---

<details>
<summary><strong>Contribution Guidelines (Expand for Details)</strong></summary>

<p>We appreciate your contribution to aibrix! To ensure a smooth review process and maintain high code quality, please adhere to the following guidelines:</p>

<h3>Pull Request Title Format</h3>
<p>Your PR title should start with one of these prefixes to indicate the nature of the change:</p>
<ul>
    <li><code>[Bug]</code>: Corrections to existing functionality</li>
    <li><code>[CI]</code>: Changes to build process or CI pipeline</li>
    <li><code>[Docs]</code>: Updates or additions to documentation</li>
    <li><code>[API]</code>: Modifications to aibrix's API or interface</li>
    <li><code>[CLI]</code>: Changes or additions to the Command Line Interface</li>
    <li><code>[Misc]</code>: For changes not covered above (use sparingly)</li>
</ul>
<p><em>Note: For changes spanning multiple categories, use multiple prefixes in order of importance.</em></p>

<h3>Submission Checklist</h3>
<ul>
    <li>[ ] PR title includes appropriate prefix(es)</li>
    <li>[ ] Changes are clearly explained in the PR description</li>
    <li>[ ] New and existing tests pass successfully</li>
    <li>[ ] Code adheres to project style and best practices</li>
    <li>[ ] Documentation updated to reflect changes (if applicable)</li>
    <li>[ ] Thorough testing completed, no regressions introduced</li>
</ul>

<p>By submitting this PR, you confirm that you've read these guidelines and your changes align with the project's contribution standards.</p>

</details>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,15 +3,26 @@
 Welcome to AIBrix, the foundational building blocks for constructing your own GenAI inference infrastructure. AIBrix offers a cloud-native solution tailored to meet the demands of enterprises aiming to deploy, manage, and scale LLMs efficiently.
 
 ## Key Features
-TODO
+
+- High density Lora management
```
Comment (Jeffwan) [inline README.md:7]: /cc @kr11 @brosoul @varungup90 please help review this part. what's the right description? any feedbacks?

File: README.md [hunk 2]
```diff
@@ -3,15 +3,26 @@
...
+- Intelligent and LLM specific routing strategies
+- LLM tailored pod autoscaler
+- AI runtime sidecar (metrics merge, fast model downloading, admin operations)
```
Comment (brosoul) [inline README.md:10]: > fast 

If we mention **fast** here, we may need to provide a comparison of its performance with some tools. But I don't think our current implementation (reusing SDK) has any reason to be faster.
The automatic selection of the number of download threads may be a breakthrough point. @Jeffwan

File: README.md (hunk 1 above)
Comment (kr11) [reply README.md:7]: @Jeffwan  Here are some drafted features about autoscaling, for your reference:
- **Direct Metrics Access**: Optimize scaler responsiveness with direct metrics retrieval from vLLM engine servers, bypassing traditional data pathways.
- **Multiple Autoscaling Algorithms**: Equip your infrastructure with HPA and KPA to ensure scalable and adaptable LLM deployment.
- **Autoscaling Case Study**: Provide actionable insights by examining autoscaling performance from experiments on practical LLM applications.

Comment (Jeffwan) [reply README.md:7]: For detailed features, we can list them in the feature intro section once this https://github.com/aibrix/aibrix/issues/2 is done

Comment (Jeffwan) [reply README.md:7]: here, we just a short description. any comments here?

File: README.md (hunk 2 above)
Comment (Jeffwan) [reply README.md:10]: Great. It's ok that we list some of the ***expected*** features here. We can adjust to more concrete words before release

**Pull request merged**